projects/addon-doc/src/components/demo/demo.component.ts

```diff
@@ -164,4 +176,8 @@ export class TuiDocDemoComponent implements OnInit {
     private getUrlTree(): UrlTree {
         return this.urlSerializer.parse(this.locationRef.path());
     }
+
+    private get params(): Params | TuiDemoParams {
```

Comment:
Looks like `Params` is irrelevant here still.

Reply:
But I need for typing
<img width="717" alt="image" src="https://user-images.githubusercontent.com/12021443/232515825-cf5fcb66-2821-45a9-a1e0-04fe8336a8c1.png">

instead of
<img width="583" alt="image" src="https://user-images.githubusercontent.com/12021443/232516005-93899cba-852f-412b-80d8-96655c1ca635.png">